config/ion_auth.php

```diff
@@ -100,9 +100,9 @@
 $config['hash_method']			= 'bcrypt';	// bcrypt, argon2, or argon2id
 $config['bcrypt_default_cost']		= 10;		// Set cost according to your server benchmark - but no lower than 10 (default PHP value)
```

Comment:
Is it worth making this:

`defined(PASSWORD_BCRYPT_DEFAULT_COST) ? PASSWORD_BCRYPT_DEFAULT_COST : 10;`